packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -3,20 +3,38 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-alert-background-color: Specifies the background color of the component.
+ * @prop --calcite-alert-border-color: Specifies the border color of the component.
+ * @prop --calcite-alert-chip-background-color: Specifies the background color of the queue count chip in the component.
+ * @prop --calcite-alert-chip-border-color: Specifies the border color of the queue count chip in the component.
+ * @prop --calcite-alert-chip-corner-radius: Specifies the corner radius of the queue count chip in the component.
+ * @prop --calcite-alert-chip-text-color: Specifies the color of the queue count chip text in the component.
+ * @prop --calcite-alert-close-background-color-active: Specifies the background color of the close button in the component when the close button is active.
+ * @prop --calcite-alert-close-background-color-focus: Specifies the background color of the close button in the component when the close button is focused.
+ * @prop --calcite-alert-close-background-color-hover: Specifies the background color of the close button in the component when the close button is hovered.
+ * @prop --calcite-alert-close-background-color: Specifies the background color of the close button in the component.
+ * @prop --calcite-alert-close-icon-color-active: Specifies the color of the close button icon in the component when the close button is active.
+ * @prop --calcite-alert-close-icon-color-focus: Specifies the color of the close button icon in the component when the close button is focused.
+ * @prop --calcite-alert-close-icon-color-hover: Specifies the color of the close button icon in the component when the close button is hovered.
+ * @prop --calcite-alert-close-icon-color: Specifies the color of the close button icon in the component.
+ * @prop --calcite-alert-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-alert-link-text-color: Specifies the color of the link text in the component.
+ * @prop --calcite-alert-message-text-color: Specifies the color of the message text in the component.
+ * @prop --calcite-alert-shadow: Specifies the shadow of the component.
+ * @prop --calcite-alert-status-color: Specifies the status color of the component.
```

Comment:
Maybe calcite-alert-accent/highlight-color? This is tied to the `kind` property and not status.

Notice uses `border` - but I think we should change that to accent/highlight as well, and set `notice-border` to none so these can be set independently. Thoughts?

Reply:
I like the idea of using `accent` to generalize names. Would this properly describe `kind`, `status` and similar props?

Reply:
Yeah, I think so - in this case the accent is different than the border, I think that wouldn't always be the case (for instance, border-color can represent a selected tab state, that's fine). There are components that have two borders - one "normal border" and one that represents either a selected state, or in this / Notice case, the kind-determined accent / highlight.

Reply:
If we make the change here I can make one in Notice as well to match, basically adding a second css prop for accent.

Reply:
I like accent color but can we run it by @ashetland and @SkyeSeitz?

Reply:
I like `accent` as well. Moving Notice away from `border` to `accent` would leave border open in the future if Notice ever got a visual update that included a traditional border of some kind. (That's a lot of borders right there, apologies for that.)